src/background_thread.c

```diff
@@ -364,7 +364,11 @@ background_thread_create(tsd_t *tsd, unsigned arena_ind) {
 	pre_reentrancy(tsd);
 	int err;
 	load_pthread_create_fptr();
-	if ((err = pthread_create(&info->thread, NULL,
+	/*
+	 * To avoid complications (besides reentrancy), create internal
```

Comment:
Right. That's why I was trying to get "the bottom of the stack" symbol, but didn't find a way to do so. __real_pthread_create doesn't look widely available. Then I didn't find a way with dlopen / dlsym there.